_opensearch/rest-api/index-apis/dangling-index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Dangling indices
+parent: index-apis
+grand_parent: REST API reference
+nav_order: 84
+---
+
+# Dangling indices API
```

Comment:
if indices is hard coded, ok

but currently we are changing any text "indices" -> "indexes" throughout.